spec/models/user_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe User, type: :model do
+  let (:account) {FactoryGirl.create :account}
+  let (:email) {Faker::Internet.email}
+  let (:password) {Faker::Alphanumeric.alpha(number: 10)}
+  let (:first_name) {Faker::Name.first_name}
+  let (:last_name) {Faker::Name.last_name}
+  it "is valid with attributes" do
+    user = account.users.create(email: email, password: password, first_name: first_name, last_name: last_name)
```

Comment:
The user should be created from its model instead of being created from an association of another model (account.users)
`user = User.create(...attrs)`